components/src/preact/lineageFilter/lineage-filter.tsx

```diff
@@ -85,15 +90,32 @@ const LineageFilterInner: FunctionComponent<LineageFilterInnerProps> = ({
 
     return (
         <>
-            <input
-                type='text'
-                class='input input-bordered w-full'
-                placeholder={placeholderText !== undefined ? placeholderText : lapisField}
-                onInput={onInput}
-                ref={inputRef}
-                list={lapisField}
-                value={initialValue}
-            />
+            <div className='relative w-full'>
+                <input
+                    type='text'
+                    class='input input-bordered w-full pr-10'
+                    placeholder={placeholderText !== undefined ? placeholderText : lapisField}
+                    onInput={onInput}
```

Comment:
```suggestion
                    onInput={() => onInput(inputRef.current?.value === '' ? undefined : inputRef.current?.value)}
```

And change line 68 from
```typescript
    const onInput = () => {
        const value = inputRef.current?.value === '' ? undefined : inputRef.current?.value;
``` 
to
```typescript
    const onInput = (value?: string) => {
```